BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift

```diff
@@ -54,7 +54,8 @@ final class DigitalInvoiceCoordinator: Coordinator {
         let extractionResult = digitalInvoice.extractionResult
         if let skontoDiscounts = try? SkontoDiscounts(extractions: extractionResult),
            GiniBankConfiguration.shared.skontoEnabled {
-            let skontoViewModel = SkontoViewModel(skontoDiscounts: skontoDiscounts)
+            let skontoViewModel = SkontoViewModel(skontoDiscounts: skontoDiscounts,
+                                                  isWithDiscountSwitchAvailable: true)
```

Comment:
[suggestion] @ValentinaIancu-Gini after renaming to "isWithDiscountSwitchAvailable" here should be `false`